doc/fips/SONiC-OpenSSL-FIPS-140-3.md

```diff
@@ -163,6 +180,20 @@ ENABLE_FIPS ?= n
 ```
 If the ENABLE_FIPS_FEATURE is not set, then the option ENABLE_FIPS is useless.
 
+## SONiC FIPS Command lines
+### The command line to enable or disable FIPS
+sonic-installer set-fips <image> [--enable-fips|--disable-fips]
```

Comment:
`set-fips --enable-fips` feels redundant compared to just `set-fips --enable` but this is just nitpicking.
If the plan is to have future knobs to this `set-fips` command then it's probably better to keep it the way you currently have it.

Reply:
@Staphylo , thanks for your comment.
The option --enable-fips can be skipped, the option --enable-fips is not necessary.
To enable it:
```
sonic-installer set-fips
```
To disable it:
```
sonic-installer set-fips --disable-fips
```